docs/06-concepts/18-testing/01-get-started.md

```diff
@@ -124,10 +124,14 @@ volumes:
 3. Create a `test.yaml` file and add it to the `config` directory:
 
 ```yaml
-# This is the configuration file for your local test environment. All ports are set to zero in this file which makes the server find the next available port. This is needed to enable running tests concurrently.
+# This is the configuration file for your test environment.
+# All ports are set to zero in this file which makes the server find the next available port.
+# This is needed to enable running tests concurrently. To set up your server, you will
 # need to add the name of the database you are connecting to and the user name.
 # The password for the database is stored in the config/passwords.yaml.
-
+#
+# When running your server locally, the server ports are the same as the public
+# facing ports.
```

Comment:
Question: What does this addition mean? This isn't really clear to me and might confuse a reader of the documentation as well.

Reply:
This was copy paste from the other config files, but you're right, this doesn't make sense for the test file. Will remove!

Reply:
e2f78d2